config_examples/template_config.yaml

```diff
@@ -45,24 +43,94 @@ telescope:
+            fill_value: extrapolate
+            # if you want to use a polynomial interpolator instead, then
+            # interpolator: poly1d
+            # kwargs not accepted for this; see numpy.poly1d documentation
+    Trx: 0.0 # receiver temperature for thermal noise
+defaults:
+    # behavior of specifying default config and some other
+    # parameters (e.g. omega_p, Tsky_mdl) currently unknown
+    default_config: 'h2c'
 systematics:
-    # TODO: fill out what should go in here
-analysis:
-    # TODO: fill out what should go in here
-simulation:
+    # XXX: perhaps change include section of simulation to just list rfi
+    # and include all the specified rfi effects if rfi is listed
+    rfi:
+        # see hera_sim.rfi documentation for details on parameter names
+        rfi_stations:
+            stations: !!null
+        rfi_impulse:
+            chance: 0.001
+            strength: 20.0
+        rfi_scatter:
+            chance: 0.0001
+            strength: 10.0
+            std: 10.0
+        rfi_dtv:
+            freq_min: 0.174
+            freq_max: 0.214
+            width: 0.008
+            chance: 0.0001
+            strength: 10.0
+            strength_std: 10.0
+    sigchain:
+        gains:
+            gain_spread: 0.1
+            dly_rng: (-20, 20)
+        # XXX: the gen_reflection_gains function is somewhat difficult
+        # to work with in a general way at the moment; this should be
+        # addressed in the future, since there currently is not a neat
+        # way of handling this from a configuration file
+        reflections: # TODO: make a YAML constructor for this?
+            amp: !!null
+            dly: !!null
+            phs: !!null
+    crosstalk:
+        # only one of the two crosstalk methods should be specified
+        whitenoise_xtalk:
+            amplitude: 3.0
+        # cross_coupling_xtalk:
+            # the following parameters are all floats, but it's a little
+            # unclear what the best way to handle this for an array is;
+            # the current implementation would treat all antennas the same
+            # amp: !!null
+            # dly: !!null
+            # phs: !!null
+sky:
+    Tsky_mdl: !Tsky
+        # TODO: add entry for directory the file is stored in
+        # need to update YAML constructor for this functionality
+        # the current assumption is that the file is in the hera_sim/data
+        # folder if an absolute path is not specified
+        datafile: HERA_Tsky_Reformatted.npz
+        # interp kwargs are passed to scipy.interp.RectBivariateSpline
+        interp_kwargs: {}
+        # TODO: add functionality for handling multiple polarizations
+        # this involves updating both the YAML constructor and Simulator
+        #
+    # TODO: investigate how to properly handle EoR here, since this
+    # won't work without modifying some things. 
     eor:
         noiselike_eor:
             eor_amp: 0.00001
             min_delay: !!null
```

Comment:
`None`; trying to specify a value as `None` by writing something like `parameter: None`, then it'll actually be interpreted as `'None'`

Reply:
right. Can you just omit `min_delay` here (since I assume its default is None)?

Reply:
I can, but I wrote the template so that places where function parameters are defined (like here) lists all of the parameters in the signature with their default values. So removing something because its default is the same as what it's set to in the template config doesn't make much sense to me.

Reply:
OK, yup, that makes sense.